models/model.js

```diff
@@ -23,28 +23,54 @@ exports.fetchArticleById = (article_id) => {
 };
 
 exports.updateArticleVoteById = (articleVoteUpdate, article_id) => {
- console.log(articleVoteUpdate, article_id)
+  console.log(articleVoteUpdate, article_id);
   return db
     .query(
       `UPDATE articles 
         SET votes = votes + $1
         WHERE article_id = $2 
         RETURNING *;`,
-      [articleVoteUpdate, article_id],
-     
+      [articleVoteUpdate, article_id]
     )
     .then(({ rows }) => {
-      console.log(rows[0])
       return rows[0];
-    })
+    });
 };
 
 exports.fetchUsers = () => {
-  return db
-    .query(`SELECT * FROM users;`).then((result) => {
-      console.log(result.rows)
-      return result.rows;
-    })
-}
+  return db.query(`SELECT * FROM users;`).then((result) => {
+    return result.rows;
+  });
+};
 
+exports.fetchArticles = (
+  sort_by = "created_at",
+  orderQuery = "DESC"
+  
+) => {
+  const order = orderQuery.toUpperCase()
+  const greenList = ["created_at", "topic", "votes"]
 
+  if(!greenList.includes(sort_by) ) {
+    console.log("bad sortby")
+    return Promise.reject({ status: 400, msg: "Bad request"})
+  }
+  
+  if(order !== "DESC" && order !== "ASC") {
+    console.log("bad order")
+    return Promise.reject({ status: 400, msg: "Bad request"})
+  }
```

Comment:
You could group these checks together as the resulting behaviour is the same - responding with a 400 bas request